### ISSUE TITLE
New Example Config

### ISSUE THREAD

**Issue by Vortigern-The-Grey** [catppuccin/waybar#11]
Seeing as the old example config is broken, could it be replaced by a working one with similar aesthetics?

**Comment by rubyowo** [comment]
You're free to open a PR!

**Comment by BlitDev** [comment]
> You're free to open a PR!

Yeah, someone definitely needs to make a new config. Now I'm trying to revive the old one on Hyprland, maybe later I'll make PR, but I don't understand why blocks are not colored in different colors. I guess it's necessary to set the colors manually, as well as actually create blocks, because on the screenshot they are only on the left side of the workspaces.

@rubyowo Can you please explain how to do this (if you're the creator of the preview)?

![image](https://github.com/user-attachments/assets/39219922-fe07-4c6c-b7f2-3afb372b2422)

**Comment by rubyowo** [comment]
The preview image is actually an image formed by adding 4 different screenshots of the bar using different catppuccin flavors with [catwalk](https://github.com/catppuccin/catwalk)
